app/routes/forum/components/ThreadDetail.tsx

```diff
@@ -0,0 +1,80 @@
+import { Flex, LoadingIndicator } from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import moment from 'moment';
+import { Link, useParams } from 'react-router-dom';
+import { fetchThread } from 'app/actions/ForumActions';
+import { CommentView } from 'app/components/Comments';
+import { Content, ContentMain } from 'app/components/Content';
+import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
+import { selectCommentsByIds, selectThreadsById } from 'app/reducers/threads';
+import { useUserContext } from 'app/routes/app/AppRoute';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import type Comment from 'app/store/models/Comment';
+import type { DetailedThread } from 'app/store/models/Forum';
+
+const ThreadDetail = () => {
+  const { threadId } = useParams<{ threadId: string }>();
+  const dispatch = useAppDispatch();
+  const user = useUserContext();
+  usePreparedEffect(
+    'fetchDetailForum',
+    () => threadId && dispatch(fetchThread(parseInt(threadId ?? '0'))),
+    [threadId]
+  );
+
+  const thread: DetailedThread = useAppSelector((state) =>
+    selectThreadsById(state, { threadId })
+  );
+
+  const comments: Comment[] = useAppSelector((state) =>
+    selectCommentsByIds(state, thread ? thread.comments : [])
+  );
+
+  const fetchingThreads = useAppSelector((state) => state.threads.fetching);
+  const fetchingComments = useAppSelector((state) => state.comments.fetching);
+
+  return (
+    thread && (
+      <LoadingIndicator loading={fetchingThreads && fetchingComments}>
```

Comment:
No need to show loading indicator if threads or comments is not empty in the redux store. Consider using skeleton components <3

Reply:
Okay I'll have a look after the other things are resolved on how to implement this.